src/SAWScript/Parser.y

```diff
@@ -319,7 +321,7 @@ instance Show ParseError where
   show e =
     case e of
       UnexpectedEOF     -> "Parse error: unexpected end of file"
-      UnexpectedToken t -> "Parse error at " ++ show (tokPos t) ++ ": Unexpected `" ++ tokStr t ++ "'"
+      UnexpectedToken t -> "Parse error at " ++ show (tokPos t) ++ ": Unexpected `" ++ (unpack $ tokStr t) ++ "'"
```

Comment:
A minor stylistic thing, but I would prefer:

```suggestion
      UnexpectedToken t -> "Parse error at " ++ show (tokPos t) ++ ": Unexpected `" ++ unpack (tokStr t) ++ "'"
```

Same amount of parentheses, but without the need to mix in another infix operator between the `++`s.